core/app/jobs/spree/themes/screenshot_job.rb

```
@@ -0,0 +1,70 @@
+        query = "#{BASE_SCREENSHOT_API_URL}?token=#{screenshot_api_token}&url=#{url}&output=#{SCREENSHOT_OUTPUT}&file_type=#{SCREENSHOT_FILE_TYPE}&retina=true&enable_caching=true"
+
+        # Send a GET request to the API
+        uri = URI.parse(query)
+        response = Net::HTTP.get_response(uri)
+
+        case response.code
+        when '301', '302'
+          redirect_uri = response['location']
+          redirect_uri = URI.parse(redirect_uri)
+          response = Net::HTTP.get_response(redirect_uri)
+          save_screenshot(theme, response)
```

RuboCop (1.73)

[warning] Remove debugger entry point `save_screenshot(theme, response)`.

(Lint/Debugger)